run.sh

```diff
@@ -6,8 +6,10 @@ set -x
 
 BUCKET=$1
 THEME=$2
+RELEASE_DATA=$3
```

Comment:
maybe the order RELEASE_DATA, BUCKET, THEME makes more sense than BUCKET, THEME, RELEASE_DATA?

Reply:
tested, otherwise LGTM; will need to update docs to describe inputting command to AWS Batch

Reply:
Updated